### PR TITLE
make lint pass, and add it to travis build

### PR DESCRIPTION
**Pull request by meriadec**
lint, lint, i love linting things.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,17 +15,41 @@
...
+    {
+      "name": "Loëck Vézien",
+      "email": "loeck@mailjet.com",
+      "url": "github.com/lohek"
+    },
+    {
+      "name": "Rémi Parmentier",
+      "email": "rparmentier@tilt-studio.fr",
+      "url": "github.com/rparmentier"
+    }
```
Comment (meriadec) [inline package.json]: a bit verbose but this is how npm reindent package.json after a dep save

**Comment by iRyusa** [comment]
Yai ! :+1: for lint

**Comment by GuillaumeBadi** [comment]
:+1:

File: package.json (hunk 1 above)
Comment (GuillaumeBadi) [inline package.json]: You might as well add your name/email/url @meriadec

Comment (meriadec) [inline package.json]: ye, when I'll make a "real" contribution :) I mean, on teh core

**Pull request merged**